app/src/main/java/com/tohami/newsapi/ui/news/NewsActivity.java

```diff
@@ -0,0 +1,108 @@
+package com.tohami.newsapi.ui.news;
```

Comment:
Structure of the packages should be as follows:
ui package to include the following:

- Base

- main

- News listing

- News details

- Navigation for side menu

Reply:
need discuss

Reply:
done